### ISSUE TITLE
Autosubmit and the GUI stopped without no apparent reason

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#863]
In GitLab by @montsecostasuros on Aug 31, 2022, 10:20

The experiment a54n was running smoothly, but today when I checked the status, it was stopped without apparent reason.
The autosubmit GUI showed:
![Screenshot_from_2022-08-31_09-27-50](https://earth.bsc.es/gitlab/es/autosubmit/uploads/8ffdff01d53479be34e9bf0a41c687c9/Screenshot_from_2022-08-31_09-27-50.png)

However, I could see in the LOGs that the CMORTM5, CMORATM and SIM3 are already finished:

`-rw-r--r-- 1 pr1enx12 pr1enx00            0 Aug 30 18:23 a54n_19900101_fc00_1_CMORTM5_COMPLETED
-rw-r--r-- 1 pr1enx12 pr1enx00            0 Aug 30 18:38 a54n_19900101_fc00_1_CMORATM_COMPLETED
-rw-r--r-- 1 pr1enx12 pr1enx00            0 Aug 30 20:26 a54n_19900101_fc00_3_SIM_COMPLETED`

Moreover, @mcastril found an alert:

```Job Name                           Job Id         Job Status     Job Platform        Job Queue      
2022-08-30 18:23:10,557 a54n_REMOTE_SETUP                  24984880       COMPLETED      marenostrum4        prace          
2022-08-30 18:23:10,557 a54n_19900101_fc00_INI             24985224       COMPLETED      marenostrum4        prace          
2022-08-30 18:23:10,557 a54n_19900101_fc00_1_SIM           24985228       COMPLETED      marenostrum4        prace          
2022-08-30 18:23:10,557 a54n_19900101_fc00_2_SIM           24986318       COMPLETED      marenostrum4        prace          
2022-08-30 18:23:10,557 a54n_19900101_fc00_1_SAVEIC        24986317       COMPLETED      marenostrum4        prace          
2022-08-30 18:23:10,557 a54n_19900101_fc00_1_CMORTM5       24988536       RUNNING        marenostrum4        debug          
2022-08-30 18:23:10,557 a54n_19900101_fc00_3_SIM           24988537       QUEUING        marenostrum4        prace          
2022-08-30 18:23:10,558 a54n_19900101_fc00_1_CMORATM       24988538       QUEUING        marenostrum4        debug          
2022-08-30 18:23:10,558 a54n_19900101_fc00_2_SAVEIC        24988535       QUEUING        marenostrum4        prace          
2022-08-30 18:23:10,560 Saving JobList: /esarchive/autosubmit/a54n/pkl/job_list_a54n.pkl
2022-08-30 18:23:10,564 Job list saved
2022-08-30 18:23:10,655 Updating FAILED jobs
2022-08-30 18:23:10,656 Updating WAITING jobs
2022-08-30 18:23:10,656 Update finished
```

I've rerun the autosubmit expid, and now it is continuing here:
![Screenshot_from_2022-08-31_10-22-27](https://earth.bsc.es/gitlab/es/autosubmit/uploads/9fb258fc085c57530bb96672064442ef/Screenshot_from_2022-08-31_10-22-27.png)

@dbeltrankyl, could you have a look? I want to prevent this from happening again.

Note: this experiment is a test, and I'm writing many variable results in the .out log and creating huge files. Could this be the reason for autosubmit stopping?

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Aug 31, 2022, 10:36

Hello, 

I see that you are using the 3.13.0 version, and this suddenly stop seems strongly related to mem issues that should've been fixed on 3.14.0b one month ago.

The issue is the large out/err files that aren't accepted in 3.13.0 due this mem issue

**Comment by kinow** [comment]
In GitLab by @mcastril on Sep 1, 2022, 10:26

Thanks @dbeltrankyl can you check if it was fixed in 3.14.0 too ?
